src/components/Reportees/index.jsx

```diff
@@ -1,56 +1,78 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { withStyles } from '@material-ui/core/styles';
-import DrilldownIcon from '../DrilldownIcon';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Paper from '@material-ui/core/Paper';
+import CONFIG from '../../config';
 
-const styles = ({
+const styles = theme => ({
   root: {
-    margin: '0 0.5rem 0 0',
-  },
-  header: {
-    margin: '0.5rem 0 0.5rem 0',
-  },
-  icon: {
-    fontSize: '1rem',
-    verticalAlign: 'bottom',
-  },
-  person: {
-    // This makes it line up better with the table of components
-    padding: '1px',
-    display: 'flex',
+    width: '50%',
+    marginTop: theme.spacing.unit * 3,
+    overflowX: 'auto',
   },
 });
 
 const sortByPersonName = (a, b) => (a.cn <= b.cn ? -1 : 1);
 
 const Reportees = ({
-  classes, ldapEmail, partialOrg, onPersonDetails,
+  classes, ldapEmail, partialOrg, metrics,
 }) => (
-  <div className={classes.root}>
-    <div height="1rem">&nbsp;</div>
-    {Object.values(partialOrg)
-      .filter(({ cn }) => cn !== ldapEmail)
-      .sort(sortByPersonName)
-      .map(({ cn, mail }) => (
-        <div key={mail} className={classes.person}>
-          <DrilldownIcon
-            name={mail}
-            onChange={onPersonDetails}
-            properties={{
-              ldapEmail: mail,
-            }}
-          />
-          <span>{`${cn} `}</span>
-        </div>
-      ))}
-  </div>
+  <Paper className={classes.root}>
+    <Table className={classes.table}>
+      <TableHead>
+        <TableRow>
+          <TableCell />
+          {Object.values(CONFIG.reporteesMetrics).map(({ label }) => (
+            <TableCell key={label}>{label}</TableCell>
+          ))}
+        </TableRow>
+      </TableHead>
+      <TableBody>
+        {Object.values(partialOrg)
+          .filter(({ cn }) => cn !== ldapEmail)
+          .sort(sortByPersonName)
+          .map(({ cn, mail, bugzillaEmail }) => (
+            <TableRow key={mail}>
+              <TableCell className={classes.leftColumn} key={mail}>{`${cn} `}</TableCell>
+              {Object.keys(CONFIG.reporteesMetrics).map((metricUid) => {
+                const countLink = ((metrics || {})[bugzillaEmail] || {})[metricUid];
+                return (
+                  <TableCell align="right" key={metricUid}>
+                    {countLink && (
+                      <a
+                        key={countLink.link}
+                        href={countLink.link}
+                        target="_blank"
+                        rel="noopener noreferrer"
+                        className={classes.link}
+                      >
+                        {countLink.count}
+                      </a>
+                    )}
+                  </TableCell>
```

Comment:
![Screen Shot 2019-04-15 at 8 46 37 AM](https://user-images.githubusercontent.com/3766511/56133905-7882a000-5f5b-11e9-8ec3-eac762ca7832.png)

The numbers are too far from the column name.

Reply:
I've addressed this:
<img width="309" alt="image" src="https://user-images.githubusercontent.com/44410/56135710-512dd200-5f5f-11e9-8902-139b21b57e0f.png">